microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -60,10 +61,49 @@ The Microsoft Edge team will independently decide on MV3 migration timelines for
 | TBD | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge, by using enterprise policies. |
 | TBD | No change. | Manifest V2 extensions will no longer function in Microsoft Edge, even with the use of enterprise policies. |
 
-Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.
+<!-- todo: existing MV2 extensions already published on the Microsoft Edge Add-ons website can be updated until the end of FY-24. [what Month/Day?]  This timeline is subject to change. -->
```

Comment:
```suggestion
Existing MV2 extensions already published on the Microsoft Edge Add-ons website can be updated until December 31, 2024.  This timeline is subject to change.
```
@sivMSFT - add this statement?  if so, this statement should be moved into the summary table.  is date correct?